actions/user_actions.jsx

```diff
@@ -697,3 +697,17 @@ export async function sendPasswordResetEmail(email, success, error) {
         error({id: err.server_error_id, ...err});
     }
 }
+
+/**
+ * Send a getUserAnalytics request for a user (userId) and team id (teamId)
+ * Pass a string indicating which dataset to return (name)
```

Comment:
It's not clear to me from this comment what name is. Since the parameters are all documented below (and it _is_ clear what name is there), maybe chance the summary comment to "Send a getUserAnalytics request to fetch the named type of analytics for the given user's interactions with the given team"?